Felienne/hedy
# [BUG FIX] Saved teacher adventures are loaded correctly again

**Description**
In this PR we fix an issue where saved teacher adventures where not loaded properly. This was due to a check in the `tabs.ts` code that verified is something was a teacher adventures, in that case: change save name and return. However, the loading of the saved program code in the editor was **below** this return statement, which was never reached when on a teacher adventure tab. This bug is due to the complex structure for which we load and switch tabs, the teacher adventures are handled as a separate adventure. We should create a separate issue to clean the tabs up a bit.

**Fixes**
This PR fixes #2354.

**How to test**
Testing is quite complex as it is best to first encounter the bug to understand the issue:

- Go to either Alpha or production
- Create a class, make sure you have a student account enrolled
- Create a teacher adventure, customize the class and include this adventure
- As the student -> open the adventure and save the program
- When loading verify that the code editor is empty

**Now**
 Repeat the steps above, but verify that the saved program is now indeed loaded correctly in the editor.
